ibis/backends/pyspark/compiler.py

```diff
@@ -775,11 +775,11 @@ def compile_clip(t, op, **kwargs):
 
     def column_min(value, limit):
         """Return values greater than or equal to `limit`."""
-        return F.when(value < limit, limit).otherwise(value)
+        return F.when((value < limit) & ~F.isnull(value), limit).otherwise(value)
```

Comment:
Is this codecov warning accurate? It looks to me like this should get hit every time `compile_clip()` is called. Is `compile_clip()` not getting called at all?

Reply:
It's probably not accurate.